### PR TITLE
GOPATH is a multi-value environment variable

### LINKED ISSUE #285: GOPATH is a multi-value environment variable
I see you catch an error by this message:

```
2018/06/22 23:28:20 kubebuilder must be run from the project root under $GOPATH/src/package.
```

When `GOPATH=/Users/abd/go`, it works. and when `GOPATH=/Users/abd/v1/go:/Users/abd/v2/go`, it does not work.

I assume you didn't consider the multi-value aspect of `GOPATH`.

Indeed, you can append multiple values to `GOPATH` by separating them through `:`.

### PR DESCRIPTION
**Pull request by luisdavim**
This should fix #285

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -21,10 +21,10 @@ import (
 	"log"
 	"os"
 	"path/filepath"
-	"strings"
 
 	"github.com/spf13/cobra"
 
+	toolsUtil "sigs.k8s.io/controller-tools/pkg/crd/util"
```
Comment (DirectXMan12) [inline cmd/main.go]: we don't generally use camelCase package names (the general practice is for all-lower-case package names, which makes them a bit easier to distinguish when reading code).

Comment (luisdavim) [reply cmd/main.go]: Thanks for the review, I've changed it to `toolsUtil `

**Comment by DirectXMan12** [comment]
looks like you've got an unused variable (`goSrc`).

**Comment by DirectXMan12** [comment]
the OSX job gave output (looks like one of your tests doesn't pass).  Looks like the issue on the linux test came from dep solving for too long.

**Comment by luisdavim** [comment]
I don't know what's wrong with the tests now, they do pas on my laptop. It seems to be some issue with `dep`...

**Comment by droot** [comment]
Travis tests have been super flaky past couple of days. I retriggered and this time it passed.

**Pull request merged**